templates/history-widget.template.html

```diff
@@ -6,19 +6,21 @@ <h2 class="history__title">Transaction History</h2>
   </p>
 
   <div class="history__table" ng-if="widget.payments.length > 0">
-    <p ng-if="widget.hideSmallAmounts">
-      <small>
-        This tool is hiding payments smaller than 0.01 XLM.
-        <button ng-click="widget.toggleDisplaySmallAmounts()">View All</button>
-      </small>
-    </p>
+    <div ng-if="widget.showSmallAmountsToggle()">
+      <p ng-if="widget.hideSmallAmounts">
+        <small>
+          This tool is hiding payments smaller than 0.5 XLM. <br />
```

Comment:
Can we make the value dynamic (`MINIMUM_AMOUNT_TO_DISPLAY`)?

Reply:
👍 Done